packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -68,6 +70,28 @@ describe('Sequence - ', () => {
     });
   });
 
+  it('user-defined Send', async () => {
+    const send: Send = (response, result) => {
+      response.setHeader('content-type', 'text/plain');
+      response.end(`CUSTOM FORMAT: ${result}`);
+    };
+    app.bind('send').to(send);
+
+    const client = await whenIMakeRequestTo(app);
+    await client.get('/name').expect('CUSTOM FORMAT: SequenceApp');
+  });
+
+  it('user-defined Reject', async () => {
+    const reject: Reject = (response, request, error) => {
+      response.statusCode = 418; // I'm a teapot
```

Comment:
I just meant that you'd want the response body to contain some sort of message, or something to that effect, rather than just 418, but it doesn't *actually* matter since it's just a test of the ability to define your rejection handler.